cats/models.py

```diff
@@ -0,0 +1,99 @@
+"""
+Models used in app
+"""
+import datetime
+
+from django.core.exceptions import ValidationError
+from django.db import models
+
+STATUS_CHOICES = [
```

Comment:
You can use models.IntegerChoices (or just enum) here. Its more legible and you can use constant values from that choices in other places like your RentalAdmin.actions. 
Something like that:
queryset.update(status=Status.Pending)